deploy/helm/README.md

```diff
@@ -0,0 +1,157 @@
+# swo-k8s-collector
+
+Assets to monitor Kubernetes infrastructure using [SolarWinds Observability](https://documentation.solarwinds.com/en/success_center/observability/default.htm#cshid=gh-k8s-collector)
+
+## Table of contents
+
+- [About](#about)
+- [Installation](#installation)
+- [Limitations](#limitations)
+- [Customization](#customization)
+
+Development documentation: [development.md](doc/development.md)
```

Comment:
Not sure if this should be in the Helm chart readme. Also, the link path is now broken.

Reply:
It gets to package:
```
ll swo-k8s-collector                                                                                                [add-readme-to-helm-package]
total 600
-rw-r--r--   1 jaroslav.fedor  staff    15B Sep 11 16:42 .helmignore
-rw-r--r--   1 jaroslav.fedor  staff    52K Sep 11 16:42 CHANGELOG.md
-rw-r--r--   1 jaroslav.fedor  staff   251B Sep 11 16:42 Chart.lock
-rw-r--r--   1 jaroslav.fedor  staff   598B Sep 11 16:42 Chart.yaml
-rw-r--r--   1 jaroslav.fedor  staff   7.3K Sep 11 16:42 README.md
drwxr-xr-x   3 jaroslav.fedor  staff    96B Sep 11 16:42 charts
-rw-r--r--   1 jaroslav.fedor  staff    13K Sep 11 16:42 events-collector-config.yaml
-rw-r--r--   1 jaroslav.fedor  staff    45K Sep 11 16:42 metrics-collector-config.yaml
-rw-r--r--   1 jaroslav.fedor  staff    26K Sep 11 16:42 node-collector-config.yaml
drwxr-xr-x   6 jaroslav.fedor  staff   192B Sep 11 16:42 proto
-rw-r--r--   1 jaroslav.fedor  staff    57K Sep 11 16:42 swo-k8s-collector-3.2.0-eks.1.tgz
drwxr-xr-x  30 jaroslav.fedor  staff   960B Sep 11 16:42 templates
-rw-r--r--   1 jaroslav.fedor  staff    45K Sep 11 16:42 values.schema.json
-rw-r--r--   1 jaroslav.fedor  staff    28K Sep 11 16:42 values.yaml
```

Reply:
But links does not work. Probably it will be better to copy readme in help package during release process.